exampleCourse/questions/templateFixedMultipleChoice/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "2dafe1f8-7b11-4efd-8d24-77b6b457b2e9",
+    "title": "Template pl-multiple-choice Fixed: Select one answer from a fixed set of responses",
```

Comment:
Can we have different types of MC question randomization:
1. One correct answer, many incorrect answers, everything hard-coded.
2. Multiple correct hard-coded answers. E.g., "Which of the following is an even number" as MC.
3. Randomization of the question prompt, selected from a list (this is the "planets" question).
4. Randomization of the question prompt using random numbers (like "demoRandomMultipleChoice").